recipes/mender-monitor/debian-1.1.x/control

```diff
@@ -0,0 +1,19 @@
+Source: mender-monitor
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-monitor
+Architecture: all
+Depends: bash, lmdb-utils, mender-auth | mender-client (>=2.5.0), ${misc:Depends}
+Description: Mender Monitor client
+ Mender Monitor is a Mender add-on which enables the reporting of certain
+ device events to the Mender server, such as log events, service files statuses
+ and D-Bus signals.
+ 
```

Comment:
just for the record, this is what we say in the docs:
```
The Mender Monitor add-on gives you instant visibility into issues within your device fleet. In addition, it allows you to customize the conditions to detect when certain parts of your infrastructure malfunction or need your attention.

To conserve bandwidth and facilitate edge processing, Mender Monitor optimizes data transmission by only sending data when specific conditions are met, ensuring efficient data utilization. Compared to classic approaches of sending all the data about all the nodes to the server all the time, which leads to massive bandwidth savings and allows instant local processing.
```
this is not a reuqest for changes, just to double check if we are aligning docs and deb descriptions. feel free to ignore.